### DISCUSSION TITLE
Sidebar pages name

### DISCUSSION THREAD

**Discussion by Shubham-EV** [fuma-nama/fumadocs#827] [Q&A] [answered]
Is there any way to change the name of a page in the sidebar? As of now, the name of pages is based on the frontmatter of the page. For example, I want the name of a page to be "Introduction" in the sidebar, but the frontmatter title of the page is different.

**Comment by fuma-nama** [comment]
You can add another frontmatter property, like `pageTitle` and use it instead in your `page.tsx`. 
Or you can remove `<DocsTitle>` component and simply add the titles to your documents directly.

**Comment by Shubham-EV** [comment]
Can't we do like this? This would be more convenient way?

```json
{
  "title": "Name of Folder",
  "pages": ["guide", "components"]
}
```

something like

```json
{
  "title": "Name of Folder",
  "pages": [{
     "guide": "Guide",
     "components": "Components",
  }]
}
```

**Reply by fuma-nama** [reply]
Honestly I don't think it is better, I hated the time when I need to type the title twice for `meta.json` and the MDX file.

Plus adding a new frontmatter property wouldn't take much time

**Reply by Shubham-EV** [reply]
> Plus adding a new frontmatter property wouldn't take much time

can you tell me how to do it? Please refer me link of it in docs

**Reply by fuma-nama** [reply]
It's after the release of Fumadocs MDX v10 so you probably missed it:
https://fumadocs.vercel.app/docs/mdx/configuration#extend-schema

**Reply by Shubham-EV** [reply]
Is it not possible in previous version the one in which `source.ts` consists of `source: createMDXSource(map)`??

**Reply by fuma-nama** [reply, marked as answer]
you can too, but the public docs no longer list it: https://fumadocs-4wsin8ko7-sonmoosans-projects.vercel.app/docs/mdx/validation#example
